xbmc/music/MusicDatabase.cpp

```diff
@@ -11919,7 +11912,7 @@ void CMusicDatabase::ExportToXML(const CLibExportSettings& settings,
               }
             }
             xmlDoc.Clear();
-            TiXmlDeclaration decl("1.0", "UTF-8", "yes");
+            // TiXmlDeclaration decl("1.0", "UTF-8", "yes");
```

Comment:
What's up with this?

Reply:
It is a repeated definition, initialised so far up in the code I wanted to have a comment as a reminder of how it is set. I guess better to add a comment that looks less like code